src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (firstNumber) => (secondNumber) => {
+    count += 1;
+
+    if (count > 3 && count % 2 === 0) {
+      return 'Bzzz... Error!';
+    } else {
+      return firstNumber + secondNumber;
+    }
```

Comment:
If we write `return` inside `if` , `else` is redundant.
```suggestion
    if (count > 3 && count % 2 === 0) {
      return 'Bzzz... Error!';
    } 
    
    return firstNumber + secondNumber;
```